packages/typescriptlang-org/src/components/search/ResultRow.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { useState } from "react"
+import { FormattedRelativeTime } from "react-intl"
+
+import { cx } from "../../lib/cx"
+import { SearchHit } from "./types"
+
+import "./ResultRow.scss"
+
+export type ResultRowprops = {
+  exactMatch?: boolean
+  installer: [string, string]
+  hit: SearchHit
+}
+
+export const ResultRow: React.FC<ResultRowprops> = ({
+  exactMatch,
+  installer,
+  hit: {
+    description,
+    downloadsLast30Days,
+    humanDownloadsLast30Days,
+    modified,
+    name,
+    repository,
+    types,
+  },
+}) => {
+  const npmUrl = `https://www.npmjs.com/package/${name}`
+  const [icon, label, viaUrl] =
+    types.ts === "included"
+      ? ["in", "included", repository?.url || npmUrl]
+      : [
+          "dt",
+          "from Definitely Typed",
+          `https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/${name}`,
```

Comment:
Question: is this an assumption we can make? _(I haven't seen any indication otherwise)_

Reply:
Close, it needs to handle `@absinthe/socket-apollo-link types` -> `absinthe__socket-apollo-link`

https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/absinthe__socket-apollo-link